src/api/app/controllers/webui/projects/meta_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Webui
+  module Projects
+    class MetaController < WebuiController
+      require_dependency 'opensuse/validator'
+      before_action :set_project, only: [:show]
+      before_action :set_project_by_name, only: [:update]
```

Comment:
Hm, what is the main difference? That set_project_by_name also respects remote projects, right? Maybe we should reflect that in the name? Because otherwise it is not clear why there are two different before filters...

Reply:
@ChrisBr please suggest a better name.

Reply:
```set_project_by_name``` is the same name used in the ```ProjectController``` so to keep the same name makes easier to identify a possibility to refactor it to use the same method in both. If you have a better name, please share with me.

Reply:
I think it is not about the name, but about having two actions that seem to be doing the same :thinking:

Reply:
I just tried it and it seems to work with ``set_project`` as well. Which seems to be logical because why should ``show`` only work for local projects and ``update`` works for local and remote that doesn't make sense. So please change to:

```
before_action :set_project
```

And drop the ``only`` and ``set_project_by_name`` part.